core/deployment/src/main/java/io/quarkus/deployment/JniProcessor.java

```diff
@@ -29,9 +29,9 @@
         /**
          * @deprecated JNI is always enabled starting from GraalVM 19.3.1.
          */
-        @Deprecated
-        @ConfigItem(defaultValue = "true")
-        boolean enable = true;
+
+        @ConfigItem
+        boolean enable;
```

Comment:
This changes the default value from `true` to `false`; we don't want that! I'd suggest to leave this one alone; it was correct already.